synapse/lib/cell.py

```diff
@@ -1741,6 +1744,114 @@ async def _addAdminUser(self, username):
     async def initServiceEarly(self):
         pass
 
+    async def initCellStorage(self):
+        self.drive = await s_drive.Drive.anit(self.slab, 'celldrive')
+        self.onfini(self.drive.fini)
+
+    async def addDriveItem(self, info, path=None, reldir=s_drive.rootdir):
+
+        iden = info.get('iden')
+        if iden is None:
+            iden = info['iden'] = s_common.guid()
```

Comment:
Doesn't look like `iden` gets used, was this intended to check if the drive already has the item to avoid pushing a nexus entry?

Reply:
Nope, just vestigial. Thanks!